### PR TITLE
Fix upgradegui english

### PR DESCRIPTION
**Pull request by kevdagoat**
See the comments for suggestions on what to add/do

### CODE REVIEW

File: decompressed/gui_file/usr/share/transformer/scripts/upgradegui [hunk 1]
```diff
@@ -133,6 +133,7 @@ UPGRADE_FROM_STABLE=0
 overlay_available_space=$(df /overlay | sed -n 2p | awk {'{print $3}'})
 if [ $overlay_available_space -lt 3000 ]; then
 	logger_command "ERROR: not enough space to complete operation" >&2
+	#Should probably implement some sort of erase here <kevdagoat>
```
Comment (kevdagoat) [inline decompressed/gui_file/usr/share/transformer/scripts/upgradegui]: Suggestions?

File: decompressed/gui_file/usr/share/transformer/scripts/upgradegui [hunk 2]
```diff
@@ -144,7 +145,7 @@ if [ ! -f $WORKING_DIR/$FILE_NAME ]; then #Check if file exist as offline upload
 	logger_command "Downloading GUI file..."
 	# Enter /tmp folder
 	if ! cd "$WORKING_DIR"; then
-	logger_command "ERROR: can't access $WORKING_DIR" >&2
+	logger_command "ERROR: Can't access $WORKING_DIR" >&2
 	ubus send fwupgrade '{"state":"done"}'
```
Comment (kevdagoat) [inline decompressed/gui_file/usr/share/transformer/scripts/upgradegui]: Call a handler here to say fw upgrade failed? eg. ubus send fwupgrade '{"state":"failed"}'

File: decompressed/gui_file/usr/share/transformer/scripts/upgradegui [hunk 3]
```diff
@@ -156,7 +157,7 @@ if [ ! -f $WORKING_DIR/$FILE_NAME ]; then #Check if file exist as offline upload
 	
 	# Download new GUI to /tmp
 	if ! $curl $URL_BASE/$FILE_NAME --output $WORKING_DIR/$FILE_NAME; then
-	logger_command "ERROR: can't find new GUI file" >&2
+	logger_command "ERROR: Can't find new GUI file" >&2
```
Comment (kevdagoat) [inline decompressed/gui_file/usr/share/transformer/scripts/upgradegui]: Call a handler here to say fw upgrade failed? eg. ubus send fwupgrade '{"state":"failed"}'

File: decompressed/gui_file/usr/share/transformer/scripts/upgradegui [hunk 4]
```diff
@@ -174,11 +175,12 @@ if [ ! -f $WORKING_DIR/$FILE_NAME ]; then #Check if file exist as offline upload
 		rm $WORKING_DIR/$CHECKSUM_FILE
 		logger_command "ERROR: File corrupted!!!" >&2
 		logger_command "ERROR: Removing upgrade file and aborting." >&2
+		#Call a handler here to say fw upgrade failed? eg. ubus send fwupgrade '{"state":"failed"}'
```
Comment (kevdagoat) [inline decompressed/gui_file/usr/share/transformer/scripts/upgradegui]: I already did this one :O

File: decompressed/gui_file/usr/share/transformer/scripts/upgradegui [hunk 5]
```diff
@@ -194,9 +196,10 @@ if [ ! -f $WORKING_DIR/$FILE_NAME ]; then #Check if file exist as offline upload
 			# Download stable GUI to /tmp
 			
 			if ! $curl $URL_BASE/$STABLE_FILE_NAME --output $WORKING_DIR/$STABLE_FILE_NAME; then
-				logger_command "ERROR: can't find stable GUI file" >&2
+				logger_command "ERROR: Can't find stable GUI file" >&2
 				rm $WORKING_DIR/$FILE_NAME
 				rm $WORKING_DIR/$CHECKSUM_FILE
+				#Call a handler here to say fw upgrade failed? eg. ubus send fwupgrade '{"state":"failed"}'
```
Comment (kevdagoat) [inline decompressed/gui_file/usr/share/transformer/scripts/upgradegui]: And this

File: decompressed/gui_file/usr/share/transformer/scripts/upgradegui (hunk 3 above)
Comment (bovirus) [reply decompressed/gui_file/usr/share/transformer/scripts/upgradegui]: I prefer to leave in lowercase "can't"

File: decompressed/gui_file/usr/share/transformer/scripts/upgradegui (hunk 2 above)
Comment (bovirus) [reply decompressed/gui_file/usr/share/transformer/scripts/upgradegui]: I prefer to leave in lowercase "can't".
For me it's better to leave untouch as much as possible the English strings in te esource.

**Pull request merged**